# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: build/rust/env_logger/lib.rs
# Commit: 00d82acfaa2f
# Full Hash: 00d82acfaa2f56095e5033a6905427d43b95eccb
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r 4ba88410cdf6 -r 00d82acfaa2f build/rust/env_logger/lib.rs
--- a/build/rust/env_logger/lib.rs	Fri Jan 27 18:35:14 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,5 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-pub use env_logger::*;